server/actions_spec_revisions.go

```diff
@@ -0,0 +1,294 @@
+package server
```

Comment:
We need to include license headers in all new files (sorry I'm just now noticing this)

Reply:
Will do.